qcelemental/util/importing.py

```diff
@@ -40,12 +48,20 @@ def which(command, *, return_bool=False):
     bool
         When `return_bool=True`, returns whether or not found.
 
+    Raises
+    ------
+    ModuleNotFoundError
+        When `raises_error=True` and command not found.
+
     """
-    lenv = {'PATH': ':' + os.environ.get('PATH') + ":" + os.path.dirname(sys.executable)}
+    lenv = {'PATH': ':' + os.environ.get('PATH', '') + ':' + os.path.dirname(sys.executable)}
     lenv = {k: v for k, v in lenv.items() if v is not None}
 
     ans = shutil.which(command, mode=os.F_OK | os.X_OK, path=lenv['PATH'])
 
+    if raise_error and ans is None:
+        raise ModuleNotFoundError(f"Command '{command}' not found in envvar PATH.")
```

Comment:
I think you convinced me of this one where we should always add custom messages. What if we added:

`"Command '{command}' not found in envvar PATH. {if install else ''}"`

or similar here? Would hide a bit more logic.

Reply:
Where `install` is a message passed into the fn? I decided against that once, but not strenuously. Fine by me.